### PR TITLE
iputils: 20200821 -> 20210202

### PR DESCRIPTION
**Pull request by primeos**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/os-specific/linux/iputils/default.nix [hunk 1]
```diff
@@ -18,18 +18,10 @@ in stdenv.mkDerivation rec {
...
     repo = pname;
-    rev = "s${version}";
-    sha256 = "1jhbcz75a4ij1myyyi110ma1d8d5hpm3scz9pyw7js6qym50xvh4";
+    rev = version;
+    sha256 = "08j2hfgnfh31vv9rn1ml7090j2lsvm9wdpdz13rz60rmyzrx9dq3";
   };
 
-  patches = [
-    # Proposed upstream patch to reduce dependency on systemd: https://github.com/iputils/iputils/pull/297
-    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/os-specific/linux/iputils/default.nix:27]: Please remove fetchpatch from inputs.

File: pkgs/os-specific/linux/iputils/default.nix [hunk 2]
```diff
@@ -6,7 +6,7 @@
 with lib;
```
Comment (SuperSandro2000) [inline pkgs/os-specific/linux/iputils/default.nix:6]: I would prefer if the with would be removed and lib put where needed.

Comment (primeos) [reply pkgs/os-specific/linux/iputils/default.nix]: Sounds reasonable, I've made the necessary changes (but I kept it for meta, as usual).

**Comment by primeos** [comment]
I also tested building `pkgsCross.musl64.iputils` and `pkgsCross.aarch64-multiplatform.iputils`.

**Pull request merged**